### ISSUE TITLE
User inside Notebook is root and not jovyan

### ISSUE THREAD

**Issue by kimwnasptd** [kubeflow/kubeflow#2602]
Tested on latest master with image `gcr.io/kubeflow-images-public/tensorflow-1.10.1-notebook-cpu:v-base-0524257-1200`. If I open a new terminal in the notebook, then I am user root and not jovyan.

/cc @jlewi @zabbasi

**Comment by zabbasi** [comment]
@kimwnasptd  How come? can you please elaborate. 
notebook-dir is already set in Dockerfile
Also tested one of the images one more time (gcr.io/kubeflow-images-public/tensorflow-1.12.0-notebook-cpu:v-base-97b94c0-1202)  by mounting  a sample file  in /home/jovyan. When notebook is launched my sample file  showed up in the jupyter list of files
```
docker run -v test.ipynb:/home/jovyan/test.ipynb  -it -p 8888:8888 gcr.io/kubeflow-images-public/tensorflow-1.12.0-notebook-cpu:v-base-97b94c0-1202
```

**Comment by zabbasi** [comment]
@jlewi

**Comment by zabbasi** [comment]
Hold  for postsubmit test to run so that all the recently released images are tagged to v0.5.0.
